### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by zippko**
Documentation update. Added latest version of qspi.hex file, removed unused *.conf files from gamepad/microbit project.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -97,42 +121,148 @@ MVP2
...
+Gamepad
+^^^^^^^^^
+#. Connect `micro:bit v2`_.
+#. Select gamepad app.  VS Code -> nRF Connect extension -> APPLICATIONS -> Select microbit.
+#. Copy file gamepad/microbit/build/zephyr/zephyr.hex to micro:bit (acting as a removable usb device).
 
 Monitor
-"""""""""""""""""""""
+~~~~~~~~~
+* VS Code -> nRF Connect extension -> CONNECTED DEVICES -> VCOM? COM ? -> Connect to Serial Port.
```
Comment (ladivin) [inline README.rst:236]: It's always VCOM1 (VCOM0 is dedicated to the network core). The COM number is system dependent, so I would just put "VCOM1" and that's it.

File: README.rst [hunk 2]
```diff
@@ -26,8 +26,11 @@ Doom 2                  Not Tested
...
 
+.. _Visual Studio Code: https://code.visualstudio.com/download
+.. _nRF Connect for VS Code: https://www.nordicsemi.com/Products/Development-tools/nRF-Connect-for-VS-Code/Download#infotabs
+
 Table of Contents
-^^^^^^^^^^^^^^^^^^^^^^^
+-------------------------------------------------------
 
 - `Status`_
 - `Getting Started`_
```
Comment (ladivin) [inline README.rst:36]: I know that this is not in the scope of this change, but "Hardware" section of Table of Contents does not point anywhere and the link is not displayed properly as a result. I suggest just removing this from TOC until we actually have this section.

**Review by ladivin — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (zippko) [reply README.rst:236]: Updated

File: README.rst (hunk 2 above)
Comment (zippko) [reply README.rst:36]: Good catch, fixed.

**Review by ladivin — APPROVED** [review]

**Pull request merged**